drivers/ccpp/catchem_dust_wrapper.F90

```diff
@@ -210,10 +210,10 @@ subroutine catchem_dust_wrapper_run(im, kte, kme, ktau, dt, garea, land,   &
           do i=its,ite
             if(xland(i,j).lt.1.5)then ! JianHe: why not also > 0.5?
               delp = p8w(i,kts,j)-p8w(i,kts+1,j)
-              ! -- GOCART afwa dust scheme
+              ! -- FENGSHA dust scheme
               call gocart_dust_fengsha_driver(dt,&
                 chem(i,kts,j,:),rho_phy(i,kts,j), &
-                dz8w(i,kts,j),smois(i,:,j), &
+                dz8w(i,kts,j),smois(i,1,j), &
```

Comment:
Should we also allow for flexible #'s of soil levels to be passed in? (e.g., RUC with very shallow first layer)

Reply:
I'll handle this in a future PR